spec/rack/test/cookie_spec.rb

```diff
@@ -28,6 +28,31 @@
       expect(last_request.cookies).to eq({})
     end
 
+    it 'uses the first path when many paths are defiend' do
+      cookie_string = [
+        '/',
+        'csrf_id=ABC123',
+        'path=/, _github_ses=ABC123',
+        'path=/',
+        'expires=Wed, 01 Jan 2020 08:00:00 GMT',
+        'HttpOnly'
+      ].join('; ')
+      cookie = Rack::Test::Cookie.new(cookie_string)
+      expect(cookie.path).to eql '/'
+    end
+
+    it 'uses the first path when many paths are defiend' do
+      cookie_string = [
+        '/',
+        'csrf_id=ABC123',
+        'path=/',
+        'expires=Wed, 01 Jan 2020 08:00:00 GMT',
+        'HttpOnly'
+      ].join('; ')
+      cookie = Rack::Test::Cookie.new(cookie_string)
+      expect(cookie.path).to eql '/'
+    end
```

Comment:
Typo in spec name, and also a misleading name (it only provides a single path 😄), so you probably intended to write something else?

Also, prefer `eq` to `eql` in the `expect` calls, in both new tests.